pkg/coscheduling/coscheduling.go

```diff
@@ -35,6 +37,17 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler/util"
 )
 
+// Config defines the scheduling parameters for Coscheduling plugin
+type Config struct {
```

Comment:
(optional): rename to "<PluginName>Args" - which is a [naming conversion] in the upstream, and once we migration deps to 1.19, we can have a **versioned** Args (for now just leave it as unversioned).

[naming conversion]: https://github.com/kubernetes/kubernetes/blob/1afc53514032a44d091ae4a9f6e092171db9fe10/pkg/scheduler/apis/config/types_pluginargs.go#L27

Reply:
Changed as suggested.